doc/src/conf.py

```diff
@@ -364,11 +364,44 @@
 latex_engine = 'xelatex'
 latex_use_xindy = False
 latex_elements = {
-    'passoptionstopackages': r'\PassOptionsToPackage{no-math}{fontspec}',
     'fontpkg': r'''
-\setmainfont{DejaVu Serif}
-\setsansfont{DejaVu Sans}
-\setmonofont{DejaVu Sans Mono}
+\setmainfont{DejaVu Serif}[Scale=0.97]
+\setsansfont{DejaVu Sans}[Scale=0.97]
+\setmonofont{DejaVu Sans Mono}[Scale=0.94]
+\usepackage[math-style=TeX,bold-style=TeX]{unicode-math}
```

Comment:
These options are the default behavior anyhow so
```suggestion
\usepackage{unicode-math}
```